src/app/status_bar.rs

```diff
@@ -360,9 +360,9 @@ impl StatusBar {
 
 #[derive(Msg)]
 pub enum ItemMsg {
-    /// Set the color of the status bar item.
+    // Set the color of the status bar item.
```

Comment:
This will not show in the doc in this case. Please keep them as doc-comment. I explained the source of this issue in [this comment](https://github.com/antoyo/relm/issues/247#issuecomment-740586958).